packages/@webex/widget-cc-user-state/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@webex/cc-user-state",
+  "version": "1.0.0",
```

Comment:
Please add a description to all package.json files

Reply:
Added a placeholder description. Will leave the proper description for when we are actually adding the functionalities for these widgets